### PR TITLE
Getting Started > Overview docs

### PR DESCRIPTION
**Pull request by achambers**

### CODE REVIEW

File: docs/v0.5.x/index.md [hunk 1]
```diff
@@ -2,4 +2,28 @@
 title: Overview
 ---
 
-Ember CLI Deploy is your one-stop shop for awesomeness.
+## So, what is ember-cli-deploy?
+
+In short, ember-cli-deploy is a tool that allows users to deploy their ember applications, easily, flexibly and reliably.
```
Comment (lukemelia) [inline docs/v0.5.x/index.md]: "ember -> "Ember"

File: docs/v0.5.x/index.md [hunk 2]
```diff
@@ -2,4 +2,28 @@
...
+
+Previous to ember-cli-deploy, there was no one straight answer to the question - "How do I deploy my Ember app?". ember-cli-deploy strives to answer this quesiton by providing
```
Comment (lukemelia) [inline docs/v0.5.x/index.md]: s/Previous/Prior/

Comment (lukemelia) [inline docs/v0.5.x/index.md]: s/quesiton/question/

File: docs/v0.5.x/index.md [hunk 3]
```diff
@@ -2,4 +2,28 @@
...
+a standardised, flexible deployment pipeline combined with a rich eco-system of plugins that allow users to easily and flexibly deploy to any setup.
```
Comment (lukemelia) [inline docs/v0.5.x/index.md]: s/eco-system/ecosystem/

File: docs/v0.5.x/index.md [hunk 4]
```diff
@@ -2,4 +2,28 @@
...
+
+Sure you do. And so does every one else. In fact their deployment scripts probably look a lot like yours. But slightly different. Maybe with a few extra bugs.
+And highly tailored for their use case and deployment setup.
+
+The point of ember-cli-deploy is to have a highly flexible, standardised way to deploy ember applications that is community supported. It takes away the need for you to worry
+about the maintance of your deployment scripts and lets you focus on the important stuff.
+
+## How does it work?
+
+Basically, ember-cli-deploy is a pipeline on which one or more plugins are registered. You decide which plugins you would like to use and ember-cli-deploy executes each of them in order, resulting
```
Comment (lukemelia) [inline docs/v0.5.x/index.md]: Something like: You choose and configure the plugins you want to use, and ember-cli-deploy executes the pipeline with each plugin doing its thing; things like compressing or encoding files, sending these files to this server, and this file to that server, notifying your team, etc. The end result is that your application is deployed the way you want it to be, and there is no custom code to maintain!/

**Comment by achambers** [comment]
Thanks @lukemelia. Updated

**Pull request merged**